android/clientmanager/src/main/java/io/mosip/registration/clientmanager/config/SessionManager.java

```diff
@@ -2,18 +2,22 @@
 
 import android.content.Context;
 import android.content.SharedPreferences;
+
 import androidx.annotation.NonNull;
+
 import com.auth0.android.jwt.JWT;
-import io.mosip.registration.clientmanager.R;
 
 import java.util.List;
 import java.util.Map;
 
+import io.mosip.registration.clientmanager.R;
+
```

Comment:
Do we still need a session manager? We should have handled this in flutter side r8?

Reply:
Yes, managing shared preferences on android side only